alot/walker.py

```diff
@@ -92,3 +97,6 @@ def _get_next_item(self):
 
     def get_lines(self):
         return self.lines
+
+    def append(self, iterableWalker):
```

Comment:
What happens if this is called after somebody started iterating over the IterableWalker already? Is that a case we have to think about? Even if this method is trivial it might be worth it to add a docstring to mention such constrains (if there are  some).

Reply:
Good question. I'll try to test that case somehow.